packages/amplify-cli/package.json

```diff
@@ -38,24 +38,24 @@
     "amplify-provider-awscloudformation": "1.17.0",
     "amplify-util-mock": "0.3.0",
     "babel-core": "^6.26.3",
-    "chalk": "^2.0.1",
+    "chalk": "^2.4.2",
     "chalk-pipe": "^2.0.0",
     "esm": "^3.2.25",
     "figlet": "^1.2.1",
     "folder-hash": "^2.0.0",
-    "fs-extra": "^6.0.1",
+    "fs-extra": "^8.1.0",
```

Comment:
We had some issues last time when I tried upgrading fs-extra from `6.0.1` to `7.0.0`. I think it was caused by this commit https://github.com/jprichardson/node-fs-extra/commit/e340861b4f89726da55964ac16fe74e4cd4536c2. Could you make sure the function category build works both invoke and deployment

Reply:
8.1.0 was already used in the CLI, it was not an "upgrade". In the smoke test I had function category as well and deploy succeeded. Now I tested invoke and it was succeeded too.

Reply:
> 8.1.0 was already used in the CLI, it was not an "upgrade"

It was an upgrade for @aws-amplify/cli. I don't remember exactly what was the problem, but if you have tested function category deploy, we should be good